tests/mocks/mock_image_generation_task.py

```diff
@@ -6,7 +6,7 @@
 
 @define
 class MockImageGenerationTask(BaseImageGenerationTask):
-    _input: TextArtifact = field(default="input")
+    _input: TextArtifact = field(default=TextArtifact("input"))
```

Comment:
I know this is just for unit tests, but you should still use a factory here, since `TextArtifact` is not a primitive and has state that can be mutated.